packages/app/src/components/scaffolder/customScaffolderExtensions.tsx

```diff
@@ -68,15 +68,15 @@ export const LowerCaseValuePickerFieldExtension = scaffolderPlugin.provide(
 const MockDelayComponent = (
   props: NextFieldExtensionComponentProps<{ test?: string }>,
 ) => {
-  const { onChange, formData, rawErrors } = props;
+  const { onChange, formData, rawErrors = [] } = props;
```

Comment:
I did not mean to commit this.  I upgraded to `@rjsf` 5.0.2 and `length` can be undefined in the version so this will need looking at later.

I've reverted for now.